axonserver/src/main/java/io/axoniq/axonserver/access/jpa/UserRole.java

```diff
@@ -32,19 +32,27 @@
 
     private String role;
 
+    private String context;
+
     @ManyToOne
     @JoinColumn(name = "username")
     private User user;
 
 
-    public UserRole(User user, String role) {
+    public UserRole(User user, String role, String context) {
         this.user = user;
         this.role = role;
+        this.context = context;
     }
 
     public UserRole() {
     }
 
+    public UserRole(String context, String role) {
```

Comment:
is this a value object i.o. an entity?
the bi-directional reference to User is not really needed IMO.

Reply:
Deleted bi-di reference